[DOCS] Document the new sub-feature privs for Cases (#814)
elastic/security-docs: docs/getting-started/cases-req.asciidoc

[[case-permissions]]
= Cases prerequisites

//To view cases, you need the {kib} space `Read` privilege for the `Security` feature. To create cases and add comments, you need the `All` {kib}
//space privilege for the `Security` feature.

//For more information, see
//{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].

> You can create roles and define feature privileges at different levels to manage feature access in {kib}. {kib} privileges grant access to features within a specified {kib} space and full or partial access can be given. For more information, see

Reviewer: Missing comma, and possible revision to make more active.
```suggestion
You can create roles and define feature privileges at different levels to manage feature access in {kib}. {kib} privileges grant access to features within a specified {kib} space, and you can grant full or partial access. For more information, see
```

{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].

NOTE: To send cases to external systems, you need the

> https://www.elastic.co/subscriptions[appropriate license].

Reviewer: Can this provide a bit more info on the type of license, or maybe the specific privilege required (using wording from the table on the target page)? That might save the user a click, or at least help them navigate the large table.

Something like "...you need an [appropriate license](https://www.elastic.co/subscriptions) that allows detection rule external actions" (or whatever the specific priv is from the list).

Author: Completely agree with your suggestion, but am going to table this for now since I think this is the start of a larger convo.

{kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].

NOTE: To send cases to external systems, you need the
https://www.elastic.co/subscriptions[appropriate license].

You can create roles and define feature privileges at different levels to manage feature access in {kib}. {kib} privileges grant access to features within a specified {kib} space and full or partial access can be given.

When setting up feature access to cases, make sure to configure the following:

- Base privileges for the {es-sec} solution

> - Sub-feature privileges for the Cases feature

Reviewer 2: Should we indicate that this step is optional? They really only need to configure the sub-feature privileges for the Cases feature if they want it to be different than the base privilege. That might be implied though.

Author: Good point, this is an important distinction to make. I'll also include a line at the start of the topic describing what the default Case priv settings are.

- Privileges for actions and connectors

> NOTE: Privileges set at the sub-feature level will override privileges set at the base level for the {es-sec} solution. For example, if a user's role has `All` privileges for the {es-sec} solution and `None` selected for the Cases feature, they'll have access to all features within the {es-sec} app except for cases.

Reviewer 2: This is very helpful. Should we also note that if the user does not customize the sub-feature privilege the user will be granted the same access to cases? For example if they have All for Security Solution and have not customized the cases sub-feature (the toggle is disabled), the user will have All for cases.

Author: Agree that this is a good example to include. I'll likely insert it right after the line I add about the default state of case privs.

[role="screenshot"]
image::images/cases-privs.png[]

More information about the minimum privileges needed to use cases is below:

[discrete]
[width="100%",options="header"]
|==============================================

| Action      | {kib} Privileges

> | Give full access to manage cases

Reviewer 3: ```suggestion
| Give full access to manage Cases
```
(let's check cap vs. lowercase)

a|
* `All` for the for the {es-sec} solution

> * `All` for the Action and Connector feature
> 
> NOTE: Roles without `All` Action and Connector feature privileges cannot create, add, delete, or modify case connectors.

Reviewer 3: ```suggestion
* `All` for the Actions and Connectors feature

NOTE: Roles without `All` Actions and Connectors feature privileges cannot create, add, delete, or modify case connectors.
```
